include/mbedtls/md.h

```diff
@@ -49,6 +49,12 @@ typedef enum {
     MBEDTLS_MD_RIPEMD160,
 } mbedtls_md_type_t;
 
+typedef enum {
+    MBEDTLS_SIG_RSA=0,
+    MBEDTLS_SIG_ECDSA,
+    MBEDTLS_SIG_NONE,
```

Comment:
Ok, having read the other commits, I understand why you wanted to make NONE the last, but if that's a public type that won't work.